kickstart/provision_atomic.erb

```diff
@@ -18,7 +18,9 @@ bootloader --timeout=3
 text
 
 <% if @host.os.name.match /.*fedora.*/i -%>
-ostreesetup --nogpg --osname=fedora-atomic --remote=fedora-atomic --url=<%= @host.param_true?('atomic-upstream') ? "https://dl.fedoraproject.org/pub/fedora/linux/atomic/#{@host.os.major}/" : "#{@host.operatingsystem.medium_uri(@host)}/content/repo/" %> --ref=fedora-atomic/f<%= @host.os.major %>/<%= @host.architecture %>/docker-host
+# Use @host.operatingsystem.medium_uri(@host)}/content/repo/ as the URL if you
+# have set up a local installation media for Fedora
+ostreesetup --nogpg --osname=fedora-atomic --remote=fedora-atomic --url=https://dl.fedoraproject.org/pub/fedora/linux/atomic/#{@host.os.major}/ --ref=fedora-atomic/f<%= @host.os.major %>/<%= @host.architecture %>/docker-host
```

Comment:
Default to what you have here, just gives @ekohl his option to override it via a parameter on the host. Something like
```
<% real-atomic-url = @atomic-url || "https://dl.fedora ..." -%>
```
should work?